src/test/scala/com/mwz/sonar/scala/scapegoat/ScapegoatSensorSpec.scala

```diff
@@ -32,15 +39,63 @@ import org.sonar.api.batch.sensor.internal.{DefaultSensorDescriptor, SensorConte
 import org.sonar.api.batch.sensor.issue.internal.DefaultIssue
 import org.sonar.api.config.internal.MapSettings
 import org.sonar.api.rule.RuleKey
-
-import java.nio.file.{Path, Paths}
+import scalariform.ScalaVersion
 
 /** Tests the Scapegoat Sensor */
-class ScapegoatSensorSpec extends FlatSpec with SensorContextMatchers with LoneElement with OptionValues {
+class ScapegoatSensorSpec
+    extends FlatSpec
+    with MockitoSugar
+    with SensorContextMatchers
+    with LoneElement
+    with OptionValues {
   val scapegoatReportParser = new TestScapegoatReportParser()
   val scapegoatSensor = new ScapegoatSensor(scapegoatReportParser)
 
-  behavior of "Scapegoat Sensor"
```

Comment:
I think you should add this line again.

Reply:
The reason I removed it is that in my opinion, it's redundant since the name of the test already implies what is tested in this spec.

It makes sense to have it if you test multiple classes in the same file etc., but in this case I think it's just unnecessary.

```
[info] ScapegoatSensorSpec:
[info] - should ...
```

is a bit simpler and cleaner than this

```
[info] ScapegoatSensorSpec:
[info] Scapegoat Sensor
[info] - should ...
```

Reply:
Ok, that's fair enough.
But, we should be changing it in the other specs as we go.

Reply:
Yes, definitely.